analytics/analytics.go

```diff
@@ -121,85 +122,76 @@ func (a *AnalyticsRecord) GetLineValues() []string {
 }
 
 //change name - obfuscateAndDecode request
-func (a *AnalyticsRecord) ObfuscateKey(authHeaderName string, decode bool) {
```

Comment:
@tbuchaillot even those I reverted this commit, it's still showing up in the diff